forem/forem
# CSS fixes: Onboarding dark mode design fixes

## What type of PR is this? (check all applicable)


## Description

CSS fixes: Onboarding dark mode design fixes

## Related Tickets & Documents



- Closes #15286

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

1. Run this branch locally
2. Ensure your settings are configured to have the dark theme, http://localhost:3000/settings/customization
3. Navigate to onboarding, http://localhost:3000/onboarding

### Issues before this PR

![image](https://user-images.githubusercontent.com/833231/140530350-22d2e5fd-1273-47c4-aeca-8a069aa94716.png)

![image](https://user-images.githubusercontent.com/833231/140530672-f3cdc013-cae1-442e-9ccc-cc3e554f1de1.png)

This one you can read, but the styles for the cards appear to be off at the corners (open below screenshots in full screen to notice the corner issues)
| Light mode issue  | Dark mode issue |
| ------------- | ------------- |
| <img width="463" alt="Screenshot 2023-01-24 at 5 28 30 PM" src="https://user-images.githubusercontent.com/9396084/214294301-7df06650-16ec-4b89-b982-9dec9b746ff1.png"> | <img width="466" alt="Screenshot 2023-01-24 at 5 29 00 PM" src="https://user-images.githubusercontent.com/9396084/214294312-634dc947-9786-4813-81df-68e4d02f8aaf.png"> |


### Light mode vs Dark mode screenshots
| Before  | After |
| ------------- | ------------- |
| <img width="588" alt="Screenshot 2023-01-24 at 6 14 51 PM" src="https://user-images.githubusercontent.com/9396084/214295200-0711a81a-aa1f-4bdc-b012-c2f92063b6bd.png"> | <img width="590" alt="Screenshot 2023-01-24 at 6 15 41 PM" src="https://user-images.githubusercontent.com/9396084/214295209-cf8a90ae-edcb-4880-b256-898d06dd58fb.png"> |
| <img width="462" alt="Screenshot 2023-01-24 at 5 30 55 PM" src="https://user-images.githubusercontent.com/9396084/214295444-32aaad97-b527-4f40-85c0-0d87a580aaa4.png"> | <img width="461" alt="Screenshot 2023-01-24 at 5 30 02 PM" src="https://user-images.githubusercontent.com/9396084/214295439-329fdad7-76b1-46ed-a4be-c843326a9a7c.png"> 
| <img width="707" alt="Screenshot 2023-01-24 at 6 15 13 PM" src="https://user-images.githubusercontent.com/9396084/214295550-4cbc96eb-3417-419d-bb75-194845bb1f0d.png"> | <img width="700" alt="Screenshot 2023-01-24 at 6 15 55 PM" src="https://user-images.githubusercontent.com/9396084/214295561-0c0a90b2-aca5-49b7-a75c-08d1b2e36df0.png"> |
| <img width="720" alt="Screenshot 2023-01-24 at 6 15 27 PM" src="https://user-images.githubusercontent.com/9396084/214295558-e3ac44ae-8171-4a42-932c-48c3c3e7a702.png"> | <img width="710" alt="Screenshot 2023-01-24 at 6 16 05 PM" src="https://user-images.githubusercontent.com/9396084/214295564-c24feac8-5d8a-40d0-9b37-dd307281e984.png"> |

### UI accessibility concerns?

This PR makes accessibility better.

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
